service/runner.cc

```diff
@@ -346,7 +348,7 @@ attemptTaskTermination()
             cerr << "childPid_ >= 0\n";
         }
         if (!(task_.statusState == ProcessState::STOPPED
-              || task_.statusState == DONE)) {
+              || task_.statusState == ProcessState::DONE)) {
```

Comment:
Nice, we maintain comments. Put them within an if (0) to make them compile if they need to be maintained.